nengo_bones/templates.py

```diff
@@ -182,7 +169,7 @@ def add_docs_conf_py_data(data):
     ):
         warnings.warn(
             "'tagmanager_id' looks strange. It should look like "
-            "'GTM-XXXXXXX'; got %r" % (tagmanager_id,)
+            f"'GTM-XXXXXXX'; got {tagmanager_id}"
```

Comment:
In general when doing this I preferred explicitly quoting the variable, like `; got '{tagmanager_id}'`. In the more "WYSIWYG" spirit of f-strings, I think when we want qutoes around the string we should just put quotes around the string, rather than relying on `!r` (which is unfamiliar notation to a lot of people, and even if people know that that prints the repr, I suspect that most people would not be aware that that has the effect of adding quotes when applied to strings).

Reply:
The `; got '{tagmanager_id}'` approach is what I've used in personal projects too, it is also slightly one less character than `:!r`, but mostly it is obvious what's going on.

Reply:
Sure, I'm fine with that. The advantage of `!r` in my opinion is that if some other object gets in here somewhere, it makes it clear what that object is, rather than printing its string version (which might not make it clear what it is). But in this case, I think it'd be pretty hard to get some random object in here, so I'll make the change to the explicit quotes.